src/main/java/reposense/report/ReportGenerator.java

```diff
@@ -151,20 +154,24 @@ private static void analyzeRepos(String outputPath, List<RepoConfiguration> conf
 
             try {
                 GitCheckout.checkout(config.getRepoRoot(), config.getBranch());
+                analyzeRepo(config, repoReportDirectory.toString());
+            } catch (NoAuthorsFoundWithCommitsException e) {
```

Comment:
`no authors found with commits for` is a little weird. I think `no authors with commits found for %s (%s) ` may be better.